.changes/unreleased/BUG FIXES-20230412-132501.yaml

```diff
@@ -0,0 +1,6 @@
+kind: BUG FIXES
+body: 'resource/random_password: The `generateHash()` functions has been modified
```

Comment:
typo: functions -> function

Reply:
Open question: will practitioners know what "generateHash()" is? I'd suggest tailoring the message for the behavior they will receive, which I believe is that we are preventing an error with `bcrypt` by truncating that hash to 72 bytes as it was previously. I'm sure that may raise further questions though.